eplaunch/tests/utilities/energyplus

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "EnergyPlus, Version 5.9.0-deadbeef00"
```

Comment:
I created a dummy script that will just echo the version in the same format E+ does